Tests/UnitTests/PlayMode/CardSets/BlackTempleCardsGenTests.cpp

```diff
@@ -632,6 +632,50 @@ TEST_CASE("[Priest : Minion] - BT_258 : Imprisoned Homunculus")
     CHECK_EQ(opPlayer->GetHero()->GetHealth(), 28);
 }
 
+// ---------------------------------------- SPELL - PALADIN
+// [BT_292] Hand of A'dal - COST: 2
+//  - Set: BLACK_TEMPLE, Rarity: Common
+// --------------------------------------------------------
+// Text: Give a minion +2/+2. Draw a card.
+// --------------------------------------------------------
```

Comment:
You should add comments for the information of play requirements.
```suggestion
// --------------------------------------------------------
// PlayReq:
// - REQ_TARGET_TO_PLAY = 0
// - REQ_MINION_TARGET = 0
// --------------------------------------------------------    
```